src/containers/CallIn/ThankYou/ThankYou.js

```diff
@@ -92,6 +94,19 @@ class ThankYou extends Component {
             })
             // Filter out the `covid_paused` districts
             .filter(district => district && district.status === 'active')
+            .map(district =>  {
+                const hasMadeCalls = this.props.calls && this.props.calls.byId
+                if (!hasMadeCalls) {
+                    return district
+                }
+                const hasCalledThisDistrict = Object.entries(this.props.calls.byId).find((entry)=>{
+                    const districtId = entry[0]
+                    const timestamp = entry[1]
```

Comment:
Fun fact! You can do weird things in JS like: 

```
const [districtId, timestamp] = entry
return districtId === `${district.districtId}` && timestamp > callExpiry 
```

(this is just a tiny syntactic thing-- feel free to ignore)